src/components/Calendar.tsx

```diff
@@ -43,6 +43,7 @@ const Calendar: FunctionComponent<RouteComponentProps> = () => {
       CalendarAction.ReceivedAllResources,
       `/api/events?context=${ResourceKey.Events}`,
       `/api/locations?context=${ResourceKey.Locations}`,
+      `/api/equipment?context=${ResourceKey.Equipment}`,
```

Comment:
I'll have to think about this one.  I don't know if it makes sense to pull down the entire equipment list just to view the calendar.

This might be something we just pull into reservation form.  TBD.  Let's discuss.

Reply:
I agree with this, i think it should just be pulled into the EquipmentForm, not even the Reservation Form. I believe that the reservation form only needs the Quantities dictionary which contains the requested equipment quantities.